packages/frontend-v2/pages/index.tsx

```diff
@@ -1,429 +1,86 @@
+  desc: string;
+}
 
-      <h2>SearchAPI logging!</h2>
-      <div>
-        <button
-          onClick={() => {
-            testFetchCourse("CS", "2500");
-          }}
-        >
-          fetchCourse
-        </button>
-        <button
-          onClick={() => {
-            testSearchCourses("CS", 0, 9999);
-          }}
-        >
-          searchCourses
-        </button>
-      </div>
-      <br />
-      <div>
-        <h2>API Error Handling</h2>
-        <div>
-          <h3>Toasts without logging</h3>
-          <button
-            onClick={() =>
-              toast.info("Oh btw here's some info on what you were doing")
-            }
-          >
-            info
-          </button>
-          <button
-            onClick={() =>
-              toast.success("Whatever you were doing was successful")
-            }
-          >
-            success
-          </button>
-          <button
-            onClick={() =>
-              toast.warn("Whatever you were doing was kinda successful")
-            }
-          >
-            warning
-          </button>
-          <button
-            onClick={() => toast.error("Whatever you were doing failed lol")}
-          >
-            error
-          </button>
-        </div>
-      </div>
-      <div>
-        <h3>Toasts with logging</h3>
-        <button
-          onClick={() =>
-            toast.info("Oh btw here's some info on what you were doing", {
-              log: true,
-            })
-          }
-        >
-          info
-        </button>
-        <button
-          onClick={() =>
-            toast.success("Whatever you were doing was successful", {
-              log: true,
-            })
-          }
-        >
-          success
-        </button>
-        <button
-          onClick={() =>
-            toast.warn("Whatever you were doing was kinda successful", {
-              log: true,
-            })
-          }
-        >
-          warning
-        </button>
-        <button
-          onClick={() =>
-            toast.error("Whatever you were doing failed lol", { log: true })
-          }
-        >
-          error
-        </button>
-      </div>
-      <div>
-        <h2>Client Side Error Handling</h2>
-        <button
-          onClick={() => {
-            setIsClientSideError(true);
-          }}
-        >
-          Trigger a client side error
-        </button>
-      </div>
-      <div>
-        <h2>Logging</h2>
-        <button onClick={() => logger.info("Info log")}>info</button>
-        <button onClick={() => logger.debug("Debug log")}>debug</button>
-        <button onClick={() => logger.warn("Warning log")}>warning</button>
-        <button onClick={() => logger.error("Error log")}>error</button>
-      </div>
-    </>
+const InfoSection = ({ title, desc }: InfoSectionProps): JSX.Element => {
+  return (
+    <Box w="55%">
+      <Heading pt="10%" fontSize={{ desktop: "3xl", laptop: "2xl", tablet: "xl" }} color="blue.700">
```

Comment:
i found that the auto-responsiveness of the `size` prop didn't scale down too well as i decreased window size, so i opted for `fontSize`/explicit breakpoints instead

Reply:
[This](https://chakra-ui.com/docs/components/typography/heading#changing-font-size) might be the reason why